digitalocean/resource_digitalocean_droplet.go

```diff
@@ -321,27 +321,35 @@ func resourceDigitalOceanDropletRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("status", droplet.Status)
 	d.Set("locked", droplet.Locked)
 
-	if err := d.Set("volume_ids", flattenDigitalOceanDropletVolumeIds(droplet.VolumeIDs)); err != nil {
-		return fmt.Errorf("Error setting `volume_ids`: %+v", err)
+	if publicIPv4 := findIPv4AddrByType(droplet, "public"); publicIPv4 != "" {
+		d.Set("ipv4_address", publicIPv4)
+	}
+
+	if privateIPv4 := findIPv4AddrByType(droplet, "private"); privateIPv4 != "" {
+		d.Set("ipv4_address_private", privateIPv4)
 	}
 
 	if publicIPv6 := findIPv6AddrByType(droplet, "public"); publicIPv6 != "" {
-		d.Set("ipv6", true)
 		d.Set("ipv6_address", strings.ToLower(publicIPv6))
-		d.Set("ipv6_address_private", findIPv6AddrByType(droplet, "private"))
-	} else {
-		d.Set("ipv6", false)
-		d.Set("ipv6_address", "")
 	}
 
-	d.Set("ipv4_address", findIPv4AddrByType(droplet, "public"))
+	if privateIPv6 := findIPv6AddrByType(droplet, "private"); privateIPv6 != "" {
+		d.Set("ipv6_address_private", strings.ToLower(privateIPv6))
+	}
 
-	if privateIPv4 := findIPv4AddrByType(droplet, "private"); privateIPv4 != "" {
-		d.Set("private_networking", true)
-		d.Set("ipv4_address_private", privateIPv4)
-	} else {
-		d.Set("private_networking", false)
-		d.Set("ipv4_address_private", "")
```

Comment:
@TFaga Is defaulting the values for `ipv6_address` and `ipv4_address_private` to an empty string generally problematic or just this implementation? Here's a use case that seemed reasonable to me: https://github.com/terraform-providers/terraform-provider-digitalocean/issues/92

Reply:
In general no, I wouldn’t say it’s problamatic. I was aiming to simplify the code together with reading the enabled features of a droplet. But supporting the usecase in #92 for a computed property is probably fine and reasonable. Especially since that was the case now and it’d be better to not break existing functionality.

I’ll update the PR.